### DISCUSSION TITLE
Want to contribute, but may be lacking some skills

### DISCUSSION THREAD

**Discussion by The-Ladle** [ravynsoft/ravynos#434] [General]
I found this project a couple of years ago not too long after I had gotten into programming. Then, I didn't think much of it, but now I revisited it seeing that I want to contribute. I am somewhat lacking in low level development skills, but enjoy a good learning challenge. Of low level languages, I currently only know parts of Rust and C++. Of higher level languages, I know a lot more. If you have any suggestions for how to contribute or how to improve my qualifications before contributing, please let me know.

**Comment by Sahil12524** [comment]
Same here I found this project many years ago, still i am not able to contribute them

**Reply by mszoek** [reply]
If you know C++, moving to Objective-C is not too difficult. A lot of the core stuff I'm working on is in plain C or C with bits of ObjC.
